pyproject.toml

```diff
@@ -23,8 +23,8 @@ dependencies = [
     "photutils >=1.13.0",
     "pyparsing >=2.4.7",
     "requests >=2.26",
-    # "roman_datamodels>=0.22.0,<0.23.0",
-    "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
```

Comment:
Is there an advantage to this outside of a release?  Should we point this back and the git version of roman_datamodels?

Reply:
This will be changed by as soon as a rad/rdm change is made. I'd only keep this as is if we plan on making a romancal release immediately with these changes.